src/ingest_validation_tools/directory-schemas/scrnaseq.yaml

```
@@ -1,4 +1,6 @@
--
-  pattern: 'TODO'
-  description: 'Directory structure not yet specified. https://github.com/hubmapconsortium/ingest-validation-tools/issues/452'
+#scRNA-Seq description
 
+-
+  pattern: '[^/]+/[^/]+\.fastq\.gz'
```

@icaoberg -- Are you confident about this? Do we need to require in each dataset (as opposed to the submission) that there be one directory level above the `.fastq.gz`? If that's really the case, great, but I would be less surprised by either 
```suggestion
  pattern: '.+\.fastq\.gz'
```
ie, allow files with any directory structure, or
```suggestion
  pattern: '[^/]+\.fastq\.gz'
```
ie, `.fastq.gz` must be at top level.